apps/testing-input-output/src/app/counter.component.spec.ts

```diff
@@ -1,25 +1,105 @@
-import { render } from '@testing-library/angular';
+import { render, screen } from '@testing-library/angular';
 import { CounterComponent } from './counter.component';
+import userEvent from '@testing-library/user-event';
 
 describe('CounterComponent', () => {
   describe('Given an initualValue of 10', () => {
     test('Then counterValue is 10', async () => {
-      await render(CounterComponent);
+      await render(CounterComponent, {
+        componentProperties: { initialValue: 10 },
+      });
+
+      const counter = screen.getByTestId('counter');
+
+      expect(counter).toHaveTextContent('Counter: 10');
     });
 
     describe('When clicking 5 times on increment button', () => {
       test('Then counterValue is 15', async () => {
-        await render(CounterComponent);
+        await render(CounterComponent, {
+          componentProperties: { initialValue: 10 },
+        });
+
+        const counter = screen.getByTestId('counter');
+
+        const button = screen.getAllByRole('button')[0]; //increment is first button
+
+        // need to look up -> any way to do multiple clicks at once?
```

Comment:
don't think so. The test reproduce real life event and user can only click once 😇